.prow.yaml

```
@@ -1,20 +1,13 @@
 presubmits:
   - name: pre-commit
-      decorate: true
-      skip_report: false
-      always_run: true
-      context: aicoe-ci/prow/pre-commit
-      spec:
-        containers:
-          - image: quay.io/thoth-station/thoth-precommit-py38:v0.13.0
-            command:
-              - "pre-commit"
-              - "run"
-              - "--all-files"
-            resources:
-              requests:
-                memory: "500Mi"
-                cpu: "300m"
-              limits:
-                memory: "1Gi"
-                cpu: "300m"
+    decorate: true
+    skip_report: false
+    always_run: true
+    context: aicoe-ci/prow/pre-commit
```

```suggestion
    context: pre-commit
```

In open-service-group its pre-commit :sweat_smile: